From ff14cf71c89cbf4cbde9add29dcc72b0fc8cd88f Mon Sep 17 00:00:00 2001
From: Sardaruddin Syed <ssardaruddin2002@gmail.com>
Date: Sat, 30 May 2026 23:34:07 +0000
Subject: [PATCH 03769/15333] Documentation: ABI: remove outdated USB
 power/level removal notice

The sysfs power/level interface is still implemented and documented
despite the removal notice stating it would be removed after 2010.

Remove the outdated removal timeline while keeping the deprecation
notice and recommendation to use power/control instead.

Signed-off-by: Sardaruddin Syed <ssardaruddin2002@gmail.com>
Link: https://patch.msgid.link/20260530233410.1718-1-ssardaruddin2002@gmail.com
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---
 Documentation/ABI/obsolete/sysfs-bus-usb | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Documentation/ABI/obsolete/sysfs-bus-usb b/Documentation/ABI/obsolete/sysfs-bus-usb
index bd096d33fbc7..067016e62e11 100644
--- a/Documentation/ABI/obsolete/sysfs-bus-usb
+++ b/Documentation/ABI/obsolete/sysfs-bus-usb
@@ -26,6 +26,6 @@ Description:
 		initializes all non-hub devices in the "on" level.  Some
 		drivers may change this setting when they are bound.
 
-		This file is deprecated and will be removed after 2010.
+		This file is deprecated.
 		Use the power/control file instead; it does exactly the
 		same thing.
-- 
2.55.0

